### ISSUE TITLE
The "Export as HTML" option cannot be selected using keyboard only, if you are logged into "Firefox Notes"

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/notes#716]
**[Affected versions]:**
- Firefox 58.0.2 and up

**[Affected Platforms]:**
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have a Firefox profile with the latest version of the "Firefox Notes" add-on (3.0.0rc2) installed.
- You are logged into "Firefox Notes".

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites and focus the sidebar.
2. Press the "Tab" key three times and press the "Enter" key.
3. Press the "Up" key three times and observe the behavior.

**[Expected result]:**
- The "Export as HTML" option is selected.

**[Actual result]:**
- Navigation stops at the "Disable Sync" option.

**[Notes]:**
- After the "Disable Sync" option is selected the focus cannot be changed to another button using the "Arrow" keys.
- Attached a screen recording of the issue:
![rec of the issue](https://user-images.githubusercontent.com/31243259/36476232-fa26f308-1705-11e8-9c11-ae34fe298765.gif)

**Comment by vladikoff** [comment]
@sebastienbarbier this only happens after enabling sync.

Seems like a conflict of `componentHandler.upgradeAllRegistered();` and Footer.js render

**Comment by vladikoff** [comment]
@sebastienbarbier if the material menu causes too many problems we can try rewriting it for the multi-note version and slating this bug for that version and not 3.1.0

**Comment by sebastienbarbier** [comment]
I saw it and already fixed in #743, but will push a quick fix for this version 👍.

**Comment by sebastienbarbier** [comment]
Couldn't solve this ... even with some really dirty code so I suggest a more aggressive option.

**Comment by vladikoff** [comment]
Should be fixed via https://github.com/mozilla/notes/issues/748
